### ISSUE TITLE
Prefix for commands not working (java)

### ISSUE THREAD

**Issue by Nebloksiam** [Pyratron/Command-Parser#5]
I get an error message "Command '/' not found."

I've looked int the class CommandParser.java and found the method:

``` java
public final boolean parse(String input, int accessLevel) {
        if (input.equals(""))
            return false;

        //Remove the prefix from the input and trim it just in case.
        input = input.trim();
        if (!prefix.equals("")) {
            int index = input.toLowerCase().indexOf(prefix.toLowerCase());
            if (index == -1)
                return false;
            input = input.substring(index, prefix.length());
        }
        if (input.equals(""))
            return false;
```

The new input String is created from a substirng of itself by cutting of the prefix.
I've replaced the follwing line

``` java
input = input.substring(index, prefix.length());
```

with

``` java
input = input.substring(index + prefix.length());
```

then it is working for me.

Further improvement:
The prefix should be found at index 0 so you could just use the length of it..

**Comment by Cyral** [comment]
Thanks for finding this issue. I just did a clean install of Windows 10 so I don't have IntelliJ and the JDK set up, but I should be able to fix this issue tomorrow.

**Comment by Cyral** [comment]
Fixed in 1.3 :)
